WeatherToday/Sources/Data/NetworkAPI/Models/APIWeather.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  APIWeather.swift
+//  WeatherToday
+//
+//  Created by Taher on 7/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+struct WeatherApi: Weather, Decodable {
+
+    let temperature: Double
+    let humidity: Double
+
+    enum CodingKeys: String, CodingKey {
+        case main
+    }
+
+    enum MainCodingKeys: String, CodingKey {
+        case temp, humidity
+    }
+
+    init(from decoder: Decoder) throws {
```

Comment:
what happen if we don't add this and `MainCodingKeys`

Reply:
@blyscuit thank you for mentioning.. that was redundant...